doc/api-extensions.md

```diff
@@ -2129,3 +2129,7 @@ projects that are referencing that network via one of their forward zones.
 
 Existing projects that have `features.networks=true` will have `features.networks.zones=true` set automatically,
 but new projects will need to specify this explicitly.
+
+## `network_txqueuelen`
+
+Adds a `txqueuelen` key to control the txqueuelen parameter of the nic device.
```

Comment:
`nic` needs to be `NIC` to avoid failing the spellcheck test.

Reply:
Can you wrap `txqueuelen` word in backticks so it doesn't fail the spellcheck test. Thanks

Reply:
Done